.github/workflows/build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and Test
+
+on:
+ push:
+   branches: [master]
+   tags:
+ pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.7', '3.8']
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
```

Comment:
nit: We don't need to specify architecture when we are using linux-only platforms.